README.md

```diff
@@ -1,6 +1,31 @@
 [![CI Status](https://github.com/AY2021S1-CS2103T-F11-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-F11-3/tp/actions)
 
-### User stories
+![Ui](docs/images/Ui.png)
+
+## Project scope
+  
+* The project simulates an ongoing software project for a desktop application (called _ProductiveNUS_).
```

Comment:
I think we should not say that ProductiveNUS is a simulation of an ongoing project. That description suits AB3 more. I think just calling it a project is good! :)

Reply:
Amended!